ui/admin/app/controllers/scopes/scope/groups/group/add-members.js

```diff
@@ -57,4 +57,22 @@ export default class ScopesScopeGroupsGroupAddMembersController extends Controll
   async cancel() {
     await this.router.replaceWith('scopes.scope.groups.group.members');
   }
+
+  /**
+   * Calls filterBy action located in the route.
+   * @param {string} field
+   * @param {[ScopeModel]} value
+   */
+  @action
+  callFilterBy(field, value) {
```

Comment:
Do we need to call these `call<action>`? Did you think it might be confusing to just call it what it originally was and just proxy it through to the route? For example `filterBy` and then calling `this.send('filterBy')`?

Reply:
we mainly just need to call it a different name or else it causes an infinite loop and calls itself instead of the action in the route. 
ex. error msg:
![image](https://github.com/user-attachments/assets/f3e253ff-af36-4333-acfc-a4284cf7b0ba)

Reply:
Ahh I didn't know that, I assumed send would only look at routes